build_tools/kokoro/gcp_ubuntu/cmake/android/build.sh

```diff
@@ -103,4 +103,11 @@ cd build-android
   -DIREE_BUILD_COMPILER=OFF \
   -DIREE_BUILD_TESTS=ON \
   -DIREE_BUILD_SAMPLES=OFF
+
+echo "Building all for device"
+echo "------------"
 "${CMAKE_BIN}" --build . -- -k 0
```

Comment:
https://buildkite.com/iree/iree-android-arm64-v8a/builds/5827#e6a720d6-693b-485b-a6b4-576e1c760deb is running
[build_tools/cmake/build_android.sh](https://github.com/google/iree/blob/main/build_tools/cmake/build_android.sh). Should that be updated instead of (or in addition to) this build_tools/kokoro/gcp_ubuntu/cmake/android/build.sh?

Reply:
Looks like it: https://buildkite.com/iree/iree-android-arm64-v8a/builds/5831